multiversx_sdk/multisig/resources.py

```diff
@@ -60,6 +62,54 @@ def __init__(self, token_identifier: str, token_nonce: int, amount: int) -> None
         )
 
 
+class UserRole(Enum):
+    NONE = 0
+    PROPOSER = 1
+    BOARD_MEMBER = 2
+
+
+class ActionFullInfo:
+    def __init__(self, action_id: int, group_id: int, action_data: "Action", signers: list[Address]) -> None:
+        self.action_id = action_id
+        self.group_id = group_id
+        self.action_data = action_data
+        self.signers = signers
+
+    @classmethod
+    def new_from_object(cls, object: Any) -> "ActionFullInfo":
+        signers = [Address(value, LibraryConfig.default_address_hrp) for value in object.signers]
```

Comment:
This uses the globally configured HRP. I saw that the controller receives an address HRP in the constructor. An idea would be to move the conversion logic in the controller? :thought_balloon:

Reply:
Moved the logic in the controller